scripts/release/check-artifacts.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
```

Comment:
nit: should we use `-e` and `-o pipefail`

Reply:
`-o pipefail` definitely. `-e` probably, but for that I also need to reread the script with brain cells enabled.

Reply:
Ok, I enabled those now. I also eliminated some image URL constant repetition.